met-web/src/components/engagement/listing/AdvancedSearch/SearchComponent.tsx

```diff
@@ -99,12 +99,12 @@ const AdvancedSearch: React.FC<filterParams> = ({ getFilterParams }) => {
             publishedToDate: '',
         });
 
-        getFilterParams([], '', '', '', '');
+        setFilterParams([], '', '', '', '');
```

Comment:
It's nicer in JS to pass one object rather than multiple args. Like setFilterParams({ statusList: ..., created_from_date: ''', etc. }). This way we will be able to create initialFilterParams = {statusList: [], created_from_date: ''...} and just pass setFilterParams(initialFilterParams) instead of setFilterParams([], '', '', '', '');

Reply:
addressed, thanks for this.